test/server/fixtures/plans/basic.json

```diff
@@ -30,6 +32,7 @@
     "quantity": 2,
     "usage_percentage": null,
     "usage_type": "price",
+    "tier_type": "flat",
```

Comment:
It isn't required for the tests to pass, but in the interest of keeping the fixtures up to date with the API, `tier_type` should be added to all of the other fixtures as well.